easybuild/easyconfigs/m/MrBayes/MrBayes-3.2.7a-iimpi-2019a.eb

```diff
@@ -0,0 +1,27 @@
+block = 'ConfigureMake'
```

Comment:
?

Reply:
They revamped (fixed) their build scripts, directories etc. for 3.2.7, so almost nothing in that easyblock was correct anymore in neither the configure or install step. So I just opted for a plain `ConfigureMake` instead.

Reply:
oh, I just now noticed my mistake.

Reply:
I'd also think the new local variables policy would make this throw an error in the tests (@boegel?)

Reply:
Well, no, because `block` is actually a known (yet rarely used) easyconfig parameter, so `eb` wouldn't treat it as a local variable:

```
$ eb -a | grep ^block
block        List of other 'block' sections on which this block depends (only relevant in easyconfigs with subblocks) [default: None]
```

The tests didn't fail though, because there is a custom easyblock for `MrBayes`.
Any reason you're not using that one @Micket?

Reply:
@boegel Yes, the reason i wrote above :)

Reply:
@Micket I would strongly prefer that we update the easyblock... Even if it's just adding a version check and falling back to using `ConfigureMake` for recent `MrBayes` versions.

Reply:
and I who saw this as an excellent opportunity to drop an evil easyblock...

Reply:
Why is it evil?